src/main.rs

```diff
@@ -401,36 +401,31 @@ fn run() -> Result<(), failure::Error> {
     if let Some(matches) = matches.subcommand_matches("config") {
         let api_key = matches.is_present("api-key");
 
-        let mut user = GlobalUser {
-            email: None,
-            api_key: None,
-            api_token: None,
-        };
+        let mut user: GlobalUser;
 
         if !api_key {
             // Default: use API token.
             message::info("Looking to use a Global API Key and email instead? Run \"wrangler config --api-key\". (Not Recommended)");
```

Comment:
this message still feels kinda weird to me; we want people to use API token, outputting a message that suggests another method, but then tagging on the caveat of "not recommended", just feels confusing.